src/map/utils/mobutils.cpp

```diff
@@ -415,7 +415,10 @@ namespace mobutils
         uint16 sMND = GetBaseToRank(grade::GetJobGrade(PMob->GetSJob(), 7), PMob->GetSLevel());
         uint16 sCHR = GetBaseToRank(grade::GetJobGrade(PMob->GetSJob(), 8), PMob->GetSLevel());
 
-        if (PMob->GetSLevel() > 15)
```

Comment:
As a head sup to people using modified subjob lv for their monsters: that setting will no longer affect mob stats in game with this change.

However, **this change IS retail accurate** and needs to happen. This of us rolling custom need to just deal with it. :+1: